mne/io/artemis123/artemis123.py

```diff
@@ -0,0 +1,258 @@
+# Author: Luke Bloy <bloyl@chop.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import os.path as op
+import datetime
+import calendar
+
+from .utils import _load_mne_locs
+from ...utils import logger
+from ..utils import _read_segments_file
+from ..base import _BaseRaw
+from ..meas_info import _empty_info
+from ..constants import FIFF
+
+
+def read_raw_artemis123(input_fname, preload=False, verbose=None):
+    """Read Artemis123 data as raw object.
+
+    Note: This reader takes data files with the extension ``.bin`` as an
```

Comment:
`.. note::` see how we do it elsewhere. Although I don't think this even needs to have the `Note:` or `.. note::` prefix at all, so might as well remove it.